src/main/java/domain/Command.java

```diff
@@ -0,0 +1,37 @@
+package domain;
+
+public class Command {
+
+    private static final int COMMAND_MIN_SIZE_INCLUSIVE = 1;
+    private static final int COMMAND_MAX_SIZE_INCLUSIVE = 5;
+    private static final String COMMAND_SIZE_ERROR_MESSAGE = "명령은 1 ~ 5 글자여야 합니다.";
+    private static final String ALL_INCLUSIVE = "all";
+
+    private final String command;
+
+    public Command(String command) {
+        validate(command);
+        this.command = command;
+    }
+
+    private void validate(String command) {
+        if (isOutOfRange(command)) {
```

Comment:
```suggestion
        if (!isRange(command)) {
```
if문에서 ! 처리를 해주는건 어떨까요? 
메서드명이 부정이고 조건문에서도 !를 사용하여 두번 생각해야하는 번거로움이 있는것 같아요